src/PlacesAutocomplete.js

```diff
@@ -282,6 +290,8 @@ PlacesAutocomplete.propTypes = {
 };
 
 PlacesAutocomplete.defaultProps = {
+  clearItemsOnError: false,
+  onError: () => console.error('place autocomplete failed'),
```

Comment:
@cburbank now that we are tackling error handling,  let's make this default error message more descriptive! I'm thinking something like 

```javascript
onError: (status) => console.log('[react-places-autocomplete]: error happened when fetching data from Google Maps API.\nPlease check the docs here (URL_TO_GITHUB_PAGE)', status)
```

This is just an idea, feel free to come up with a better error message!!